stripe/_list_object.py

```diff
@@ -76,6 +84,9 @@ def create(
             ),
         )
 
+    @_util.deprecated(
+        "This will be removed in a future version of stripe-python. Please call the `retrieve` method on the corresponding resource directly, instead of using `create` from the list object."
```

Comment:
```suggestion
        "This will be removed in a future version of stripe-python. Please call the `retrieve` method on the corresponding resource directly, instead of using `retrieve` from the list object."
```